diff --git a/core/jni/android_hardware_Camera.cpp b/core/jni/android_hardware_Camera.cpp
index 17967e0..afcdd84 100644
--- a/core/jni/android_hardware_Camera.cpp
+++ b/core/jni/android_hardware_Camera.cpp
@@ -715,10 +715,13 @@ static void android_hardware_Camera_setMetadataCb(JNIEnv *env, jobject thiz, jbo
     sp<Camera> camera = get_native_camera(env, thiz, &context);
     if (camera == 0) return;
 
+#if 0
     if(mode == true)
         rc = camera->sendCommand(CAMERA_CMD_METADATA_ON, 0, 0);
     else
         rc = camera->sendCommand(CAMERA_CMD_METADATA_OFF, 0, 0);
+#endif
+    rc = NO_ERROR;
 
     if (rc != NO_ERROR) {
         jniThrowException(env, "java/lang/RuntimeException", "set metadata mode failed");
