diff --git a/a/include/linux/msm_rotator.h b/include/linux/msm_rotator.h
index 8a2256f..41b9804 100644
--- a/include/linux/msm_rotator.h
+++ b/include/linux/msm_rotator.h
@@ -12,6 +12,8 @@
 		_IOW(MSM_ROTATOR_IOCTL_MAGIC, 2, struct msm_rotator_data_info)
 #define MSM_ROTATOR_IOCTL_FINISH   \
 		_IOW(MSM_ROTATOR_IOCTL_MAGIC, 3, int)
+#define MSM_ROTATOR_IOCTL_BUFFER_SYNC   \
+		_IOW(MSM_ROTATOR_IOCTL_MAGIC, 4, struct msm_rotator_buf_sync)
 
 #define ROTATOR_VERSION_01	0xA5B4C301
 
@@ -21,6 +23,13 @@ enum rotator_clk_type {
 	ROTATOR_IMEM_CLK
 };
 
+struct msm_rotator_buf_sync {
+	uint32_t session_id;
+	uint32_t flags;
+	int acq_fen_fd;
+	int rel_fen_fd;
+};
+
 struct msm_rotator_img_info {
 	unsigned int session_id;
 	struct msmfb_img  src;
